modules/virt-options-configuring-mdevs.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-virtual-gpus.adoc
+
+:_content-type: REFERENCE
+[id="virt-options-configuring-mdevs_{context}"]
+= Options for configuring mediated devices
+
+There are two available methods for configuring mediated devices when using the NVIDIA GPU Operator. The method that Red Hat tests uses {VirtProductName} features to schedule mediated devices, while the NVIDIA method only uses the GPU Operator.
+
+Using the NVIDIA GPU Operator to configure mediated devices::
+This method exclusively uses the NVIDIA GPU Operator to configure mediated devices. To use this method, refer to link:https://docs.nvidia.com/datacenter/cloud-native/openshift/latest/openshift-virtualization.html[NVIDIA GPU Operator with {VirtProductName}] in the NVIDIA documentation.
+
+Using {VirtProductName} to configure mediated devices::
+This method, which is tested by Red Hat, uses {VirtProductName}'s capabilities to configure mediated devices. In this case, the NVIDIA GPU Operator is only used for installing drivers with the NVIDIA vGPU Manager. The GPU Operator does not configure mediated devices.
```

Comment:
i'm not sure if this is just GH being funky, but the diff is acting like there's a stray italics text generator character. idk where it is though, the apostrophe after `{VirtProductName}` looks correct

Reply:
Right? I couldn't figure that out either. I'll try to fix it.